SourceX/controls/devices/game_controller.cpp

```diff
@@ -1,36 +1,37 @@
 #include "controls/devices/game_controller.h"
 
 #ifndef USE_SDL1
+
+#include <cstddef>
+
 #include "controls/controller_motion.h"
 #include "controls/devices/joystick.h"
 #include "stubs.h"
 
 namespace dvl {
 
-static SDL_GameController *current_game_controller = NULL;
-static bool sgbTriggerLeftDown = false;
-static bool sgbTriggerRightDown = false;
+std::vector<GameController> *const GameController::controllers_ = new std::vector<GameController>;
 
-ControllerButton GameControllerToControllerButton(const SDL_Event &event)
+ControllerButton GameController::ToControllerButton(const SDL_Event &event)
 {
 	switch (event.type) {
 	case SDL_CONTROLLERAXISMOTION:
 		switch (event.caxis.axis) {
 		case SDL_CONTROLLER_AXIS_TRIGGERLEFT:
 			if (event.caxis.value < 8192) { // 25% pressed
-				sgbTriggerLeftDown = false;
+				trigger_left_is_down_ = false;

```

Comment:
What is the odd name styling supposed to indicate?

Reply:
Private member variable

This is Google style.

What would you like me to rename them?